src/collection-view.js

```diff
@@ -14,6 +14,11 @@ Marionette.CollectionView = Marionette.View.extend({
   // flag for maintaining the sorted order of the collection
   sort: true,
 
+  // TODO: Discuss how to handle option to disable attach/beforeAttach on child view trees.
+  // These properties are a placeholder for now.
```

Comment:
What concerns do you have? I'm thinking these flags work like a regions flags and should be the same...